### DISCUSSION TITLE
ZIP-PAB01 energy meter does not update sensor values

### DISCUSSION THREAD

**Discussion by clase30** [zwave-js/zwave-js#6649] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
ZIP-PAB01 energy meter module does not update sensor values. 

Please reopen the original discussion, because HA-core experts have provided the clue to solve this bug:

https://github.com/zwave-js/node-zwave-js/discussions/6600#discussioncomment-8114056

[...] The device only reports on Endpoint 0 and Endpoint 1... But Z-Wave JS has mapped the meter reports to endpoints 1-3. HA is never told about an endpoint 0 so it can't handle the updates. [...]

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: ...
Model name: ...
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Discussion is reopened. Let's continue there.
